addons/binding/org.openhab.binding.freebox/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="lib/freeboxos-client-0.3.4-jar-with-dependencies.jar" sourcepath="/home/sysadmin/Documents/Gaël/01.SourceSafe/10.Java/projects/freeboxos-java"/>
```

Comment:
I just had a look into this jar after I noticed that it is > 4MB, which is quite huge.
It contains all kind of stuff that should not be bundled within your binding as it is already available in the target platform. This includes e.g. apache commons, jackson, Google Guava, etc.
Could you possibly try to strip this down to what is really needed?